src/devtools/client/debugger/src/components/PrimaryPanes/Outline.js

```diff
@@ -32,6 +32,9 @@ import type { Symbols } from "../../reducers/ast";
 import type { SymbolDeclaration, FunctionDeclaration } from "../../workers/parser";
 import type { Source, Context, SourceLocation } from "../../types";
 
+const outlineHeaderHeight = 29;
+const outlineFooterHeight = 25;
```

Comment:
if it were me, i'd probably use CSS vars, but I don't feel strongly either

Reply:
it is inside Outline.js there is OutlineFilter.js. The scroll ui and outline-footer are inside outline.js, so it felt easier to define them here and control the styles through JS which will also answer your previous comment.

Reply:
got it. thanks!

Reply:
I think this is one area where BEM style naming for classes could alleviate any concerns on scope widening when doing things in CSS.

That's a discussion for another time though. This PR should certainly not be going there.